man/man8/zfs-snapshot.8

```diff
@@ -44,13 +44,19 @@
 .Ar dataset Ns @ Ns Ar snapname Ns …
 .
 .Sh DESCRIPTION
-All previous modifications by successful system calls to the file system are
-part of the snapshots.
-Snapshots are taken atomically, so that all snapshots correspond to the same
-moment in time.
+Creates a snapshot of a dataset or multiple snapshots of different
+datasets.
+.Pp
+Snapshots are created atomically.  That is, a snapshot is a consistent
```

Comment:
Thanks for the clarification and updated test.  Just a couple small mandoc formatting nits.

[new sentence, new line](https://man.openbsd.org/mandoc.1#new)
    (mdoc) A new sentence starts in the middle of a text line. Start it on a new input line to help formatters produce correct spacing.

Reply:
Thanks very much for the quick feedback. I've made and committed those changes.